modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="ols-creating-the-credentials-secret-using-cli_{context}"]
-= Creating the credentials secret using the command line
+= Creating the credentials secret using the CLI
```

Comment:
```suggestion
= Creating the credentials secret by using the CLI
```

The oc CLI? Or a specialized CLI?

Reply:
@dfitzmau the PRereqs mention it's using the OCP CLI. 

This documentation is associated with a developer preview release. OLS-978 is a task to create an attribute for the oc command line. Presently, no such attribute exists in this standalone doc set, but I plan to create one in the near future and update the usage instances at that time.

Reply:
Hi @rh-tokeefe . A different writer will need to do the merge review as I done the peer review.

Reply:
ack